src/components/collection-progress-group/components/collection-progress-status-section/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { ReactNode } from 'react'
+import { IconCollections16 } from '@hashicorp/flight-icons/svg-react/collections-16'
+import { IconCheckCircleFill16 } from '@hashicorp/flight-icons/svg-react/check-circle-fill-16'
+import ProgressBar from 'components/progress-bar'
+import s from './collection-progress-status-section.module.css'
+
+/**
+ * Renders collection progress status, in a nice little box.
+ *
+ * Without authentication,
+ * we show a "Start" CTA which links to the first tutorial in the collection.
+ *
+ * When authenticated and one or more tutorials in the collection is "complete",
+ * we show a "Continue" CTA which links to the next not-"complete" tutorial,
+ * as well as a progress bar based on the count of tutorials done.
+ *
+ * When authenticated and all tutorials in the collection are "complete",
+ * we show a "Review" CTA which links to the first tutorial in the collection.
+ */
+function CollectionProgressStatusSection({
```

Comment:
I've split out this component as the "presentation" part of `CollectionProgressGroup` (fka `CollectionMetaProgress`). We re-use this component for collection progress display in the sidebar.